src/main/java/gregtech/common/blocks/BlockGasCentrifugeCasing.java

```diff
@@ -0,0 +1,49 @@
+package gregtech.common.blocks;
+
+import gregtech.api.block.IStateHarvestLevel;
+import gregtech.api.block.VariantBlock;
+
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.IStringSerializable;
+
+public class BlockGasCentrifugeCasing extends VariantBlock<BlockGasCentrifugeCasing.GasCentrifugeCasingType> {
```

Comment:
Any Reason why this needs to be separate from BlockNuclearCasing? I would like to avoid adding an entirely new Block just for 1 block, if possible.

Reply:
I think I tried doing that once and the model wasn't working, but I did it again and it seems to work now. Technically there are now two slightly different variants of gas centrifuge columns that look identical, but I guess it's fine.